Magical Nights/items/ethereal_items.json

```diff
@@ -27,7 +27,38 @@
     "price": 0,
     "symbol": ",",
     "color": "yellow",
-    "use_action": [ { "type": "firestarter", "moves": 20 } ],
+    "use_action": [ 
+      { 
+        "type": "firestarter", "moves": 20 
+      },
+      {
+        "type": "transform",
+        "msg": "You dim the flame.",
+        "target": "finger_firelighter_off"
+      } 
+  ],
+    "flags": [ "TRADER_AVOID", "FIRESTARTER", "LIGHT_8", "FLAMING", "WATER_EXTINGUISH" ]
+  },
+  {
+    "id": "finger_firelighter_off",
+    "type": "TOOL",
+    "name": "finger firelighter (dimmed)",
+    "description": "This is a small flame you can hold in your hand.  With a thought, you can cause it to light something on fire, though it is currently dimmed.",
+    "weight": "1 g",
+    "volume": "1 ml",
+    "price": 0,
+    "symbol": ",",
+    "color": "yellow",
+    "use_action": [ 
+      { 
+        "type": "firestarter", "moves": 20 
+      },
+      {
+        "type": "transform",
+        "msg": "You brighten the flame.",
+        "target": "finger_firelighter"
+      } 
+  ],
     "flags": [ "TRADER_AVOID", "FIRESTARTER", "LIGHT_8", "FLAMING", "WATER_EXTINGUISH" ]
```

Comment:
Looks to me like you forgot to actually make the dimmed version any dimmer than the other, considering they both have LIGHT_8 (and the same flags as each other in general).

Reply:
Whoops, my bad!

Reply:
Oh one sec, I can make it a copy-from.